File: qlty-cli/tests/lang/csharp/basic.in/cyclomatic/CycloIfElseIfElse.cs
namespace Cyclomatic
{
  class CycloIfElseIfElse {
    public static void Main(string[] args) {
      int x = 1;
      if (x > 0) {
        int y = 1;
      } else if (x < 0) {
        int y = 2;
      } else {
        int y = 3;
      }
    }
  }
}

File: qlty-cli/tests/lang/swift/basic.in/cyclomatic/CycloIfElseIfElse.swift
// If-else-if-else statement cyclomatic complexity test

func cycloIfElseIfElse(a: Int, b: Int, c: Int) -> Int {
    if a > b && a > c {
        return a
    } else if b > c {
        return b
    } else {
        return c
    }
}